### DISCUSSION TITLE
Update getting started guide to include ImportMap and Turbo initalization.

### DISCUSSION THREAD

**Discussion by OLuwis** [rails/rails#51603] [Docs]
The delete section on the [guide](https://guides.rubyonrails.org/getting_started.html#deleting-an-article) doesn't work, because the guide doesn't cover the initialization of ImportMap and Turbo.

**Comment by AmandaPerino** [comment]
Thank you for letting us know @OLuwis. We will add this to our To Do list, but in the meantime it's good to know that we will have a new tutorial drafted soon as well.

**Reply by OLuwis** [reply]
Just to give more details. I was following the getting started guide, and was unable to use turbo methods for deletion, and had to find a solution on stackoverflow, indicating that should initialize them.

Apparently, my "rails new" command didn't initialized both Turbo and ImportMap, but when i did a second time it worked fine.

Don't know if it is necessary, but just for precaution i think the guide should advise about theses gems initialization, just for making sure.
